shared/templates/selinux_booleans.csv

```diff
@@ -179,14 +179,14 @@ polipo_session_users,
 polipo_use_cifs,
 polipo_use_nfs,
 polyinstantiation_enabled,
-postfix_local_write_mail_spool,
+postfix_local_write_mail_spool,enable
```

Comment:
ACK, this seems to be necessary for normal postfix operation in certain cases (per https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/SELinux_Users_and_Administrators_Guide/sect-Managing_Confined_Services-Postfix-Booleans.html ).